app/src/main/java/org/fossasia/susi/ai/MainApplication.java

```diff
@@ -23,6 +25,14 @@ public static MainApplication getInstance() {
     @Override
     public void onCreate() {
         super.onCreate();
+
```

Comment:
I don't want to remove these lines .I am just asking why extra space is used here.